source/extensions/config_subscription/grpc/delta_subscription_state.h

```diff
@@ -78,13 +78,18 @@ namespace Config {
 class DeltaSubscriptionState : public Logger::Loggable<Logger::Id::config> {
 public:
   DeltaSubscriptionState(std::string type_url, UntypedConfigUpdateCallbacks& watch_map,
-                         const LocalInfo::LocalInfo& local_info, Event::Dispatcher& dispatcher,
-                         XdsConfigTrackerOptRef xds_config_tracker);
+                         Event::Dispatcher& dispatcher, XdsConfigTrackerOptRef xds_config_tracker);
```

Comment:
I find it interesting that all the tests pass with the unified_mux implementation (no changes were needed there AFAICT). Can you double check whether that is working as expected?